src/language-services/diagnosticErrorCode.ts

```diff
@@ -0,0 +1,8 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+
+export enum DiagnosticErrorCode {
+    DuplicateIdentifier = "Error.DuplicateIndentifier",
+    ParserError = "Error.Parser",
```

Comment:
nit: PQP uses the verb Parse instead of the noun Parser, meaning the errors it throws are ParseError instead of ParserError.